atflatcontrols/atflattoolbar.pas

```diff
@@ -440,7 +450,9 @@ procedure TATFlatToolbar.PopupForDropdownClick(Sender: TObject);
 begin
   C:= Sender as TControl;
   P:= C.ClientToScreen(Point(0, C.Height));
+  {$ifdef FPC}
   C.PopupMenu.PopUp(P.X, P.Y);
+  {$endif}
```

Comment:
so won't work in delphi? bad

Reply:
Yeah. TControl.popupmenu is protected and delphi won't let you access here. Probably will need a new type with a private popupmenu added to it but wanted you to look at it too.

Reply:
Use hack like it- type TControlHack = class(TControl); TControlHack(C).PopupMenu....

Reply:
Done.